requirements/runtime.txt

```diff
@@ -1,12 +1,10 @@
 lmdb
-mmcv-full>=1.3.1
+mmcv-full>=1.3.13 # To support DCN on CPU
 opencv-python<=4.5.4.60
 # MMCV depends opencv-python instead of headless, thus we install opencv-python
 # Due to a binary compatibility bug, we limit its version before 4.5.5
 # https://github.com/opencv/opencv-python/issues/602
 # https://github.com/opencv/opencv/issues/21366
 # It seems to be fixed in https://github.com/opencv/opencv/pull/21382
 # But our test is negative.
-scikit-image
 tensorboard
-yapf
```

Comment:
Why delete these two lines?

Reply:
it seems this codebase no longer rely on scikit-image, I search `skimage`, and there is no code importing this, also CI passes. 
yapf is only a code formatter, it is not a runtime dependency.

Further, I think we need to add numpy and pillow. They are now installed as a dependency of torchvison, but we'd better to add them in requirements explicitly. I will append them and rename this PR.